### PR TITLE
Remove TODOs from metrics.proto. 

### PR DESCRIPTION
**Pull request by jsuereth**
Remaining todos are bugs against specification/proto library.

See https://github.com/orgs/open-telemetry/projects/12 for TODOs.

### CODE REVIEW

File: opentelemetry/proto/metrics/v1/metrics.proto [hunk 1]
```diff
@@ -447,7 +442,6 @@ message IntHistogramDataPoint {
   // distribution of values in the histogram is unknown and only the total count and sum are known.
 
   // explicit_bounds is the only supported bucket option currently.
-  // TODO: Add more bucket options.
```
Comment (bogdandrutu) [inline opentelemetry/proto/metrics/v1/metrics.proto:450]: This is removed.

File: opentelemetry/proto/metrics/v1/metrics.proto [hunk 2]
```diff
@@ -523,7 +517,6 @@ message HistogramDataPoint {
...
```
Comment (bogdandrutu) [inline opentelemetry/proto/metrics/v1/metrics.proto:526]: This is removed.

File: opentelemetry/proto/metrics/v1/metrics.proto (hunk 1 above)
Comment (jsuereth) [reply opentelemetry/proto/metrics/v1/metrics.proto:450]: yeah, other histogram types are tracked as bugs/oteps now.

Comment (jsuereth) [reply opentelemetry/proto/metrics/v1/metrics.proto]: I see what you meant, didn't notice the merge conflict :)

**Review by bogdandrutu — APPROVED** [review]

**Review by jmacd — APPROVED** [review]

**Pull request merged**